widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/ElementUtils.java

```diff
@@ -32,6 +32,8 @@
 import com.google.gwt.user.client.ui.VerticalSplitPanel;
 import com.google.gwt.user.client.ui.Widget;
 
+import static org.pentaho.gwt.widgets.client.utils.string.StringUtils.isEmpty;
```

Comment:
Had to use a static import because there is also a **deprecated** `StringUtils` class on the same package as the `ElementUtils` class (which is `org.pentaho.gwt.widgets.client.utils`).